test_modules/run_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# running tests
+TEST_FILES=(
+  "test_modules/test_spaceship.rb"
+  "test_modules/test_scan.rb"
+)
+EXIT_CODE=0
+for TEST_FILE in ${TEST_FILES[*]} 
+do
+  ruby $TEST_FILE
```

Comment:
Per https://stackoverflow.com/a/8795650/252627 you should be able to just do `ruby -e "require 'spaceship'"` here to achieve the same result - and avoid needing those files, but that doesn't work for me locally :/

Reply:
omg! I searched for something like this and didn't manage to find 😫